pyproject.toml

```diff
@@ -1,3 +1,10 @@
 [tool.black]
 line-length = 100
 include = '\.pyi?$'
+
+[tool.bandit]
+targets = "pennylane"
+exclude_dirs = ["tests", "pennylane/labs/tests"]
+skips = [
+    "B101", # We are okay with using asserts in source code as we do not compile optimized bytecode of pennylane
```

Comment:
I'd argue that's a "for now"
I want to remove asserts in favour of safer runtime checking in the long run. Good to know this can be easily checked by removing this tag.